src/lib/Components/App.php

```diff
@@ -19,7 +19,7 @@ class App implements Component
 
     protected $templating;
 
-    protected $title;
+    protected $pageTitle;
```

Comment:
I would have kept this property like that. In this component, `title` is fine (it's really the app title)